docs/CustomApp.md

```diff
@@ -5,7 +5,7 @@ title: "Including the Admin in Another App"
 
 # Including React-Admin In Another Redux Application
 
-The `<Admin>` tag is a great shortcut got be up and running with react-admin in minutes. However, in many cases, you will want to embed the admin in another application, or customize the admin redux store deeply.
+The `<Admin>` tag is a great shortcut to be up and running with react-admin in minutes. However, in many cases, you will want to embed the admin in another application, or customize the admin redux store deeply.
```

Comment:
I was quite tricky about this sentence, and I'm not sure this is the right way to fix this. Let me know if you want to deliver different meaning. :)